views/advanced_usage.dt

```diff
@@ -55,4 +55,27 @@ block body
 	p For adding subpackages from the registry, execute <code> dub add &lt;packagename&gt;:&lt;subpackage&gt; </code>
 
 	p If you need to compile a subpackage only, use <code> dub build :&lt;subpackage&gt; </code>
-	
\ No newline at end of file
+
+	h2#have-version Conditional compilation according to dependency presence
+
+	p Projects can have additional special dependencies for specific configurations.
```

Comment:
This is not the only way how dependencies can be optional, there is also `"optional": true` which you can put on any dependency, where a package depending on it may depend on it, causing the dependency to also load it.

Reply:
Provided code is just for example; section is trying to point out `Have_<dependency>` feature